xmodule/modulestore/draft_and_published.py

```diff
@@ -136,8 +141,11 @@ def _flag_publish_event(self, course_key):
             if bulk_record and bulk_record.active:
                 bulk_record.has_publish_item = True
             else:
+                print(f"reload damn you")
                 # We remove the branch, because publishing always means copying from draft to published
                 self.signal_handler.send("course_published", course_key=course_key.for_branch(None))
+                catalog_info = self.create_catalog_data_for_signal(course_key)
```

Comment:
This is other bit of actually important code. It can use create_catalog_data_for_signal because it inherits from the BulkOperationsMixin, which is where that method is defined. This seems to be how we handle non-bulk operations, but I couldn't figure out how to hit this code using my local Studio.